scripts/singleton/get-deploy-data.js

```diff
@@ -23,6 +28,7 @@ const deployer = tx.getSenderAddress()
 
 console.log(JSON.stringify({
   deployer: toHexString(deployer),
+  gas: gasPrice * gasLimit,
```

Comment:
'gas' is often referring to 'gas used' or oven 'gas limit', this value is 'tx fee' I assume (if there is no refund, that is)